src/config.js

```diff
@@ -7,5 +7,9 @@ module.exports = {
   baseURL: process.env.baseURL || 'http://localhost:8080',
   jwtSigningKey: new Buffer(jwtSigningKeyB64, 'base64'),
   sqlite: process.env.sqlite || './data/podverseWeb.sqlite',
-  databaseName: 'podverse'
+  databaseName: 'podverse',
+
+  // TODO: change the auth0 values to podverse.fm's clientId and domain before deployment
+  auth0ClientId: 'tTGQrl5CenMDdpzcKNpmLIginyRgBJNN',
```

Comment:
This is really the only piece that needs to change (that I know of) for having successful deployment.  Ultimately since there are other issues and non-satisfactory uses of webpack I wish we would have just templated
```
<script>
window.auth0ClientId = '{{ auth0ClientId}}';
</script>
```
with nunjucks, referenced with window config in the auth.js, and then called it a day.